test/dryrun/context_vars/context_vars_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 Red Hat, Inc.
```

Comment:
I'm never sure about when/if to *update* this date in files, but I'm fairly sure it can be set to the current date for *new* files with brand-new content.